tox.ini

```diff
@@ -16,12 +16,12 @@ envlist =
 #
 
 [testenv:check-style]
-description = check code style, e.g. with ruff
+description = check code style, e.g. with flake8
 skip_install = true
 deps =
-    ruff
+    flake8
```

Comment:
Why are you reverting the `ruff` change? PR #77 has been merged, so `ruff` is the style linter being used by `stpipe`.